src/index/test/test_vespa_indexer.py

```diff
@@ -63,7 +63,7 @@ def test_get_document_generator(
 
     schema, document_id, data = document
     assert schema == FAMILY_DOCUMENT_SCHEMA
-    assert document_id == parser_output.document_metadata.family_import_id
+    assert document_id == parser_output.document_metadata.import_id
```

Comment:
I'd be tempted to rename this do `doc_import_id` or similar to avoid ambiguity in the future.